src/platform/pdf/templates/utils.js

```diff
@@ -303,36 +295,24 @@ const generateFooterContent = async (doc, parent, data, config) => {
       right: 16,
     };
 
-    const groupingStruct = i === pages.count - 1 ? 'Struct' : 'Artifact';
-    const footer = doc.struct(groupingStruct, {
-      type: 'Pagination',
-      title: 'Footer',
-      attached: 'Bottom',
-    });
-    parent.add(footer);
+    // Only allow the last footer element to be read by screen readers
+    const footer =
+      i === pages.count - 1
+        ? doc.markStructureContent('Div')
```

Comment:
very minor: for consistency with the header, could we either change this to `Sect` or the header to `Div`? (Using `Div` is probably the most semantically correct option I think...)

Reply:
Good point. Made initial header `Div` as well. (Other headers are `Artifact`s).